<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="SoftwareManagement.sm_apt_install_download_path" skipped="0" tests="1" time="52.01" timestamp="2022-04-27T09:12:22">
	<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: TypeError: 'NoneType' object is not iterable" type="BLOCKED"/>
		<system-out>09:11:30 INFO  ==============================================================
09:11:30 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
09:11:30 INFO  Dir:   software_management_end_to_end/
09:11:30 INFO  Title: Install package to directory specified in tedge config set tmp.path
09:11:30 INFO  ==============================================================
09:11:31 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:11:31 INFO  EnvironmentC8y Setup
09:11:31 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:11:46 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:11:46 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:11:47 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:11:49 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:11:51 INFO  State of current operation: SUCCESSFUL
09:11:51 WARN  TypeError occurred while running test - 'NoneType' object is not iterable
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1292, in __call__
    self.testObj.setup()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 49, in setup
    self.assertThat(&quot;False == value&quot;, value=self.check_is_installed(&quot;rolldice&quot;))
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py&quot;, line 398, in check_is_installed
    for package in package_list:
TypeError: 'NoneType' object is not iterable
09:11:52 WARN  TypeError: 'NoneType' object is not iterable ... blocked 
09:11:52 INFO  
09:11:52 INFO  cleanup:
09:11:52 INFO  Executed sudo, exit status 0
09:11:53 INFO  Executed sudo, exit status 0
09:11:57 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:11:57 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
09:11:57 INFO  Delaying execution by 7 seconds
09:12:18 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
09:12:22 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:12:22 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:12:22 INFO  
09:12:22 INFO  Test duration: 52.01 secs
09:12:22 INFO  Test final outcome:  BLOCKED
09:12:22 INFO  Test outcome reason: TypeError: 'NoneType' object is not iterable
09:12:22 INFO  
</system-out>
	</testcase>
</testsuite>
